src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -214,10 +214,32 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.clickEditSessionButton();
         editedSession.setInstructions(new Text("Made some changes"));
         feedbackEditPage.editFeedbackSession(editedSession.getEndTime(), editedSession.getStartTime(),
-                                        editedSession.getInstructions(), editedSession.getGracePeriod());
+                                        editedSession.getInstructions(), editedSession.getGracePeriod(), true);
 
         String expectedString = "The end time for this feedback session cannot be earlier than the start time.";
         feedbackEditPage.verifyStatus(expectedString);
+
+        ______TS("test discard changes to feedback session");
+
+        // Make changes to Feedback session
+        editedSession.setStartTime(Const.TIME_REPRESENTS_NOW);
+        editedSession.setEndTime(Const.TIME_REPRESENTS_LATER);
+        editedSession.setInstructions(new Text("Made some more changes"));
+        editedSession.setGracePeriod(15);
+        feedbackEditPage.editFeedbackSession(editedSession.getStartTime(), editedSession.getEndTime(),
+                                        editedSession.getInstructions(), editedSession.getGracePeriod(), false);
```

Comment:
@VamsiSangam If you have some time, would it be possible for you to take a look at this and help out? I ask you because I have noticed that you have a knack for JavaScript debugging. Thanks in advance! :)